js/app.js

```diff
@@ -1,5 +1,6 @@
 var React = require('react');
 var Quack = require('./components/quack.jsx');
+require('../stylesheets/app.scss');
```

Comment:
really, you just require it in the js and you dont have to load it with `link rel`? weird.